llm_app/model_wrappers/huggingface_wrapper/pipelines.py

```diff
@@ -46,6 +93,23 @@ def __init__(
         self.max_new_tokens = max_new_tokens
 
     def __call__(self, text, **kwargs):
+        """
+        Run the model to complete the text.
+        Args:
+            text (str): prompt to complete.
+            return_full_text (bool, optional, defaults to True): 
+                If True, returns the full text, if False, only added text is returned. 
+                Only significant if return_text is True.
```

Comment:
Do we have `return_text` somewhere? I can't see it.

Reply:
Arguments from the HF Pipeline.__call__. I picked the most relevant ones.

Reply:
Then we should add info that other args from `pipeline` are allowed too. And possibly give a url to their docs with all allowed arguments.

Reply:
Done